lib/block-patterns/text-two-columns.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Two columns of text.
+ *
+ * @package WordPress
+ */
+
+return array(
+	'title'       => _x( 'Two columns of text', 'Block pattern title', 'default' ),
+	'categories'  => array( 'columns', 'text' ),
+	'content'     => '<!-- wp:columns {"align":"full"} -->
+	<div class="wp-block-columns"><!-- wp:column -->
+	<div class="wp-block-column"><!-- wp:spacer -->
+	<div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
+	<!-- /wp:spacer -->
+	<!-- wp:paragraph {"style":{"typography":{"fontSize":"21px"}, "color":{"text":"#000000"}}} -->
+	<p class="has-text-color" style="font-size:21px;color:#000000;"><strong>' . esc_html__( 'We have worked with:', 'default' ) . '</strong></p>
+	<!-- /wp:paragraph -->
+	
+	<!-- wp:paragraph {"style":{"typography":{"fontSize":"24px","lineHeight":"1.2"}}} -->
+	<p style="font-size:24px;line-height:1.2"><a href="https://wordpress.org">' . esc_html__( 'EARTHFUND™<br>ARCHWEEKLY<br>FUTURE ROADS<br>BUILDING NY', 'default' ) . '</a></p>
```

Comment:
@MaggieCabrera These `<br>` tags are getting escaped, so I think we'll need to escape each line individually instead: 

![Screen Shot 2021-04-05 at 9 11 16 AM](https://user-images.githubusercontent.com/1202812/113577425-3f7fe200-95ef-11eb-87bb-4997c14f506e.png)